.travis.yml

```diff
@@ -1,7 +1,14 @@
 language: ruby
 cache: bundler
-sudo: false
-rvm: 2.4
+rvm:
+  - 2.6
+  - 2.4
 
+before_install: gem update --system --no-document
 install: script/bootstrap
 script: script/cibuild
+
+env:
+  matrix:
+    - JEKYLL_VERSION="~> 3.5"
+    - JEKYLL_VERSION=">= 4.0.0.pre.alpha1"
```

Comment:
```suggestion
    - JEKYLL_VERSION=">= 4.0.0.pre.beta1"
```

Reply:
@DirtyF if you look at the CI Build logs, you'll see that Bundler has installed
`jekyll-4.0.0.pre.beta1`. (Which is expected since it is *greater than* the alpha release).

Reply:
🤦‍♂ of course 😄